st2debug/tests/integration/test_submit_debug_info.py

```diff
@@ -121,6 +117,52 @@ def test_create_archive_include_all(self):
         self.assertTrue(os.path.isdir(pack_dir))
         self.assertTrue(not os.path.exists(config_path))
 
+    def test_create_archive_include_all(self):
+        archive_path = create_archive(include_logs=True, include_configs=True,
+                                      include_content=True,
+                                      include_system_info=True)
+        extract_path = tempfile.mkdtemp()
+        self._verify_archive(archive_path=archive_path,
+                             extract_path=extract_path,
+                             required_directories=['logs', 'configs', 'content'])
+
+    def _create_config_yaml_file(self):
```

Comment:
Using a sample config file checked into fixtures meant that the paths within the config file were static. In order to make the tests pass the config file needed an entry like this when run within Travis:
`/home/travis/build/StackStorm/st2/st2debug/tests/integration/fixtures/logs/st2*.log`

As a result the test wouldn't be portable to run anywhere other then Travis. The latest change generates these paths based on where the working copy is located.